examples/launch_asha_yahpo.py

```diff
@@ -23,40 +23,40 @@
 from syne_tune import Tuner, StoppingCriterion
 from syne_tune.config_space import Domain
 
-try:
-    import matplotlib.pyplot as plt
-except ImportError:
-    pass
-
 
 def plot_yahpo_learning_curves(
     trial_backend, benchmark: str, time_col: str, metric_col: str
 ):
-    bb = trial_backend.blackbox
-    plt.figure()
-    plt.title(
-        f"Learning curves from Yahpo {benchmark} for 10 different hyperparameters."
-    )
-    for i in range(10):
-        config = {
-            k: v.sample() if isinstance(v, Domain) else v
-            for k, v in bb.configuration_space.items()
-        }
-        evals = bb(config)
-        time_index = next(
-            i for i, name in enumerate(bb.objectives_names) if name == time_col
-        )
-        accuracy_index = next(
-            i for i, name in enumerate(bb.objectives_names) if name == metric_col
+    try:
```

Comment:
Lets do it this way for now, but I think we should just import normally matplotlib in this case since the example would not run anyway.

Reply:
I think you are right. I'll change it, because right now the it just clutters the code